src/content/en/updates/2018/07/css-scroll-snap.md

```diff
@@ -0,0 +1,365 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Introduction to CSS Scroll Snap.
+
+
+{# wf_updated_on: 2018-07-25 #}
+{# wf_published_on: 2018-07-24 #}
+{# wf_tags: chrome69,css,snap,scroll #}
+{# wf_blink_components: Blink>CSS #}
+{# wf_featured_image: /web/updates/images/generic/new-in-chrome.png #}
+{# wf_featured_snippet: CSS Scroll Snap allows web developers to create well-controlled scroll experiences by declaring scroll snapping positions. This enables common UX scroll patterns without the need for JavaScript. #}
+
+# Well-Controlled Scrolling with CSS Scroll Snap {: .page-title }
+
+{% include "web/_shared/contributors/flackr.html" %}
+{% include "web/_shared/contributors/majidvp.html" %}
+
+
+<style>
+figure {
+  text-align: center;
+}
+figcaption {
+  font-size: 14px;
+  font-style: italic;
+}
+</style>
+
+
+## TL;DR {: #tldr .hide-from-toc}
```

Comment:
TL;DRs should be `h3`'s with eg: `### TL;DR {: #tldr .hide-from-toc}`

Reply:
Done